### PR TITLE
Genesis 11:9 (Required language tweaks)

### PR DESCRIPTION
**Pull request by 567Turtle**
Now the whole world had one language and a common form of speech. 

And as people journeyed eastward, they found a plain in the land of Shinar and settled there.

And they said to one another, “Come, let us make bricks and bake them thoroughly.” So they used brick instead of stone, and tar instead of mortar.

“Come,” they said, “let us build for ourselves a city with a tower that reaches to the heavens, that we may make a name for ourselves and not be scattered over the face of all the earth.”

Then the LORD came down to see the city and the tower that the sons of men were building.

And the LORD said, “If they have begun to do this as one people speaking the same language, then nothing they devise will be beyond them.

**"Come, let Us go down and confuse their language, so that they will not understand one another’s speech.”**

So the LORD scattered them from there over the face of all the earth, and they stopped building the city.

That is why it is called Babel, for there the LORD confused the language of the whole world, and from that place the LORD scattered them over the face of all the earth.

:cl:
add: You can join most jobs without knowing Sol
/:cl:

### CODE REVIEW

**Comment by projectkepler-ru** [comment]
@Cyprex  one of these days I'll get perm to add label >:(

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]
Please fixes your description and title

**Comment by 567Turtle** [comment]
> Please fixes your description and title

What's wrong with them?

**Comment by NullDagaf** [comment]
> What's wrong with them?

Doesn't follow the template, also it's basically completely unclear as to what this PR does without looking at the code changes.

**Comment by 567Turtle** [comment]
> > What's wrong with them?
> 
> Doesn't follow the template, also it's basically completely unclear as to what this PR does without looking at the code changes.

Doesn't follow the template meh meh meh meh meh

**Comment by mcmeiler** [comment]
> Doesn't follow the template meh meh meh meh meh

Following the template also makes reviews easier.

Apart from that, the Changelog part of the template actually shows up as the in-game changelog. Not adding it means players do not know these changes are added once they are merged.

File: modular_skyrat/modules/customization/modules/jobs/_job.dm
Comment (mcmeiler) [inline modular_skyrat/modules/customization/modules/jobs/_job.dm:184]: This is a bad idea, in my opinion. A virologist cannot release diseases unapproved by the CMO - How are they going to get them approved if they can't talk to one another?

Comment (567Turtle) [reply modular_skyrat/modules/customization/modules/jobs/_job.dm:184]: Dunno ask psychotic moth chick

**Comment by Cyprex** [comment]
I don't really have an opinion on whether or not we should disable language requirements, but regardless these should be done in a new modular file for our codebase.
Note that you can also set it on `/datum/job` to do it for every job.
Aside from these code changes, I'll leave the decision on whether or not this is a good idea to the other maintainers

**Comment by 567Turtle** [comment]
I don't really think modularizing this further is necessary. That would just spread out the code way too much and make maintaining this already shit code even shittier, it's commented correctly.

**Comment by mcmeiler** [comment]
> I don't really think modularizing this further is necessary.

It makes upstream merges easier, as their code won't cause merge conflicts on their own modular files.

> it's commented correctly.

If I were really pedantic, it's not even commented correctly as it lacks end comments.

All in all I still think this should be in a modular file, too, if at all.

**Pull request merged**
